shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -217,6 +217,8 @@ public class FlutterActivity extends Activity
     implements FlutterActivityAndFragmentDelegate.Host, LifecycleOwner {
   private static final String TAG = "FlutterActivity";
 
+  private boolean hasRegisteredCallback = false;
```

Comment:
nit: should this get a more specific name as to what callback has been registered for context?

Reply:
Changed to: hasRegisteredBackCallback